azure-pipelines.yml

```diff
@@ -93,6 +93,18 @@ jobs:
     #    testRunTitle: '$(Agent.OS) - $(Build.BuildNumber)[$(Agent.JobName)] - Python $(python.version)'
     #  condition: succeededOrFailed()
 
+    - bash: |
+        set -e
+        pip install .
+        FILES="tm_examples/*.py"
+        for fn in $FILES
+        do
+          echo "Processing $fn example..."
+          python $fn
```

Comment:
Can we get the coverage ?

Reply:
no, it is a script not pytest and anyway you would not get anything more then the package itself